src/dune_rules/melange/melange_stanzas.mli

```diff
@@ -22,6 +22,8 @@ module Emit : sig
   type Stanza.t += T of t
 
   val decode : t Dune_lang.Decoder.t
+
+  val target_dir : t -> dir:Path.Build.t -> Path.Build.t
```

Comment:
Is this change needed for something?

Reply:
This is used in `alias_rec.ml` and it'd create a conflict if it remained in `melange_rules`

Reply:
Would you like me to move this change to a separate PR?

Reply:
If the feature cannot be implemented without it, then we can let it stay